src/c-writer.cc

```diff
@@ -3848,20 +3850,48 @@ void CWriter::Write(const ConvertExpr& expr) {
 void CWriter::Write(const LoadExpr& expr) {
   const char* func = nullptr;
   switch (expr.opcode) {
-    case Opcode::I32Load: func = "i32_load"; break;
-    case Opcode::I64Load: func = "i64_load"; break;
-    case Opcode::F32Load: func = "f32_load"; break;
-    case Opcode::F64Load: func = "f64_load"; break;
-    case Opcode::I32Load8S: func = "i32_load8_s"; break;
-    case Opcode::I64Load8S: func = "i64_load8_s"; break;
-    case Opcode::I32Load8U: func = "i32_load8_u"; break;
-    case Opcode::I64Load8U: func = "i64_load8_u"; break;
-    case Opcode::I32Load16S: func = "i32_load16_s"; break;
-    case Opcode::I64Load16S: func = "i64_load16_s"; break;
-    case Opcode::I32Load16U: func = "i32_load16_u"; break;
-    case Opcode::I64Load16U: func = "i64_load16_u"; break;
-    case Opcode::I64Load32S: func = "i64_load32_s"; break;
-    case Opcode::I64Load32U: func = "i64_load32_u"; break;
+    case Opcode::I32Load:
```

Comment:
Can you instead wrap this case statement in `// clang-format off` `  `// clang-format on` ?  (So we can keep the brief version)

Reply:
Can do!